sos/report/plugins/powerpc.py

```diff
@@ -63,7 +63,10 @@ def setup(self):
                 "/proc/ppc64/systemcfg",
                 "/var/log/platform",
                 "/var/log/drmgr",
-                "/var/log/drmgr.0"
+                "/var/log/drmgr.0",
+                "/var/log/hcnmgr",
+                "/var/ct/IBM.DRM.stderr",
+                "/var/ct/IW/log/mc/IBM.DRM/trace*"
```

Comment:
Are these regular log files that happen to be named `trace*`, or actual tracing like what is seen in `/sys/kernel/debug/tracing/trace_pipe`?

Reply:
these are trace.sp files as below... 
trace.10.sp  trace.13.sp  trace.16.sp  trace.19.sp  trace.21.sp  trace.24.sp  trace.27.sp  trace.2.sp   trace.32.sp  trace.35.sp  trace.38.sp  trace.40.sp  trace.6.sp  trace.9.sp
trace.11.sp  trace.14.sp  trace.17.sp  trace.1.sp   trace.22.sp  trace.25.sp  trace.28.sp  trace.30.sp  trace.33.sp  trace.36.sp  trace.39.sp  trace.4.sp   trace.7.sp
trace.12.sp  trace.15.sp  trace.18.sp  trace.20.sp  trace.23.sp  trace.26.sp  trace.29.sp  trace.31.sp  trace.34.sp  trace.37.sp  trace.3.sp   trace.5.sp   trace.8.sp